generic.go

```diff
@@ -23,6 +21,8 @@ const (
 	// Work queue deduplicates scheduled keys. This is the period it waits for duplicate keys before letting the work
 	// to be dequeued.
 	workDeduplicationPeriod = 50 * time.Millisecond
+
+	metricsNamespace = "ctrl"
```

Comment:
Will it be more or less convenient if we use app name here instead?

Reply:
Oh, ok, we use app name in a label. Is it better that way? I'm curious

Reply:
When I wrote some grafana dashboards for our monitoring, I had to create a subdashboard for each component - without this it would be possible to change it all into a single dashboard with a dropdown/select instead.